src/java/org/apache/cassandra/db/compaction/OperationType.java

```diff
@@ -50,15 +50,35 @@
     VIEW_BUILD("View build"),
     INDEX_SUMMARY("Index summary redistribution"),
     RELOCATE("Relocate sstables to correct disk"),
-    GARBAGE_COLLECT("Remove deleted data");
+    GARBAGE_COLLECT("Remove deleted data"),
+    DOWNGRADE_SSTABLES("Downgrade sstables"),
+    RESTORE("Restore"),
+    // operations used for sstables on remote storage
+    REMOTE_RELOAD("Remote reload", true), // reload locally sstables that already exist remotely
+    REMOTE_COMPACTION("Remote compaction", true), // no longer used, kept for backward compatibility
+    TRUNCATE_TABLE("Table truncated"),
+    DROP_TABLE("Table dropped"),
+    REMOVE_UNREADEABLE("Remote unreadable sstables"),
```

Comment:
this one is not being used as well. keep it?

Reply:
it's used in DSE `DefaultDiskErrorHandler`. do we plan to port `ErrorHandler` to CC?

Reply:
actually, it is already there, but the used `OperationType` is UNKNOWN:
http://github.com/datastax/cassandra/blob/17056f827e8160745866b10fa257c09a83ec1509/src/java/org/apache/cassandra/db/lifecycle/Tracker.java#L341-L344

Reply:
isn't "Remote unreadable sstables" a typo? They are not remote, correct?
It should be "Remove ..." ?

Reply:
you are right